src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -77,6 +81,18 @@ public class WebView: EvasObject
         /// <since_tizen> 4 </since_tizen>
         public event EventHandler<SmartCallbackArgs> UrlChanged;
 
+        /// <summary>
+        /// Event that occurs when the context menu customized.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        //public event EventHandler<ContextMenuArgs> ContextMenuCustomize;
+
+        /// <summary>
+        /// Event that occurs when the context menu item selected.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        //public event EventHandler<ContextMenuItemArgs> ContextMenuItemSelected;
```

Comment:
Could you explain why have you commented all of this? If none of it is important, please remove those commented lines.

Reply:
I am sorry, working on it, TO DO it is